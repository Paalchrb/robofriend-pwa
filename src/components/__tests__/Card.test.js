import { shallow } from 'enzyme';
import Card from '../Card'; 
import React from 'react';


it('expect to render Card component', () => {
  const wrapper = shallow(<Card />);
  expect(wrapper).toMatchSnapshot();
})