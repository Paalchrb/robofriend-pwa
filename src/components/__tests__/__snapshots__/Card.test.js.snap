// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="card"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
